main-project/src/members/entity/guest-members.entity.ts

```diff
@@ -0,0 +1,24 @@
+import { Meeting } from 'src/meetings/entity/meeting.entity';
+import {
+  BaseEntity,
+  Column,
+  Entity,
+  JoinColumn,
+  OneToOne,
+  PrimaryGeneratedColumn,
+} from 'typeorm';
+
+@Entity('guest_members')
+export class GuestMembers extends BaseEntity {
+  @PrimaryGeneratedColumn()
+  noo: number;
```

Comment:
```suggestion
  no: number;
```

오타인지 모르겠는데 확인한번 해보시라고 남깁니다~